development/tools/jobs/kyma/acceptance_tests_test.go

```diff
@@ -0,0 +1,62 @@
+package kyma
+
+import (
+	"testing"
+
+	"github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAcceptanceJobsPresubmit(t *testing.T) {
+	// WHEN
+	jobConfig, err := tester.ReadJobConfig("./../../../../prow/jobs/kyma/tests/acceptance/acceptance.yaml")
+	// THEN
+	require.NoError(t, err)
+
+	assert.Len(t, jobConfig.Presubmits, 1)
+	kymaPresubmits, ex := jobConfig.Presubmits["kyma-project/kyma"]
+	assert.True(t, ex)
+	assert.Len(t, kymaPresubmits, 1)
+
+	actualPresubmit := kymaPresubmits[0]
+	expName := "kyma-tests-acceptance"
+	assert.Equal(t, expName, actualPresubmit.Name)
+	assert.Equal(t, []string{"master"}, actualPresubmit.Branches)
+	assert.Equal(t, 10, actualPresubmit.MaxConcurrency)
+	assert.True(t, actualPresubmit.SkipReport)
+	assert.True(t, actualPresubmit.Decorate)
+	assert.Equal(t, "github.com/kyma-project/kyma", actualPresubmit.PathAlias)
+	tester.AssertThatHasExtraRefTestInfra(t, actualPresubmit.JobBase.UtilityConfig)
+	tester.AssertThatHasPresets(t, actualPresubmit.JobBase, tester.PresetDindEnabled, tester.PresetDockerPushRepo, tester.PresetGcrPush, tester.PresetBuildPr)
+	tester.AssertThatJobRunIfChanged(t, actualPresubmit, "tests/acceptance/fix")
+	assert.Equal(t, "^tests/acceptance/", actualPresubmit.RunIfChanged)
+	assert.Equal(t, tester.ImageGolangBuildpackLatest, actualPresubmit.Spec.Containers[0].Image)
+	assert.Equal(t, "/home/prow/go/src/github.com/kyma-project/kyma/tests/acceptance", actualPresubmit.Spec.Containers[0].Args[0])
```

Comment:
You can also add test for commad equal:
`assert.Equal(t, []string{"/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"}, actualPresubmit.Spec.Containers[0].Command)`

Reply:
Done